### ISSUE TITLE
getting a message no steam overlay but i can open it using shortcut

### ISSUE THREAD

**Issue by NarimantasG** [pierre-josselin/payday-2-ultimate-trainer-6#115]

**Comment by pierre-josselin** [comment]
Is steam overlay enabled in the steam settings for payday 2?

**Comment by NarimantasG** [comment]
yes. Tho i might add some times overlay fails to work, even tho its enabled i might also add that some games compleatly fail to run if steam is left runing for too long and i would need to relaunch steam to make them work again but that did not work ether. I'm compleatly flabbergasted i usualy fix these issues myself but i dont know enough about steam overlay to do so.
